examples/confluence-reader/fetch_confluence_space.py

```diff
@@ -189,3 +192,6 @@ def run(job_input: IJobInput):
         data_file,
         confluence_reader.fetch_all_pages_in_confluence_space(parent_page_id),
     )
+
+    storage = DatabaseStorage(get_value(job_input, "storage_connection_string"))
```

Comment:
you are saving all information to one row in the database?

Reply:
Yeah. Because before it was saved to a file and I just replace the line to save to a file , to save it in DB. 
 
I supposed I was too lazy to write a for loop